### DISCUSSION TITLE
Testing 3.0

### DISCUSSION THREAD

**Discussion by spikefishjohn** [GNS3/gns3-server#2088] [General]
Is 3.0 usable for testing or should I hold off?

**Comment by grossmj** [comment]
There are still a few rough edges, you should hold off a few more weeks ;) I will let you know when alpha 1 is ready

**Comment by cristian-ciobanu** [comment]
> Is 3.0 usable for testing or should I hold off?

Version 3.0.0 alpha 1 is out.

**Reply by grossmj** [reply]
This is the first alpha so please expect bugs ;) We will try to quickly fix these and release alphas often until we are comfortable for a beta release. Also, we will post some instructions for testing 3.0, what's new etc. soon

**Comment by lurendrejer** [comment]
Will apt-get automatically install 3.0?
I have quite a few servers in place, based on the old .conf file based username and password scheme - I guess this will blow up since 3.0 has a database for these things.
